Podfile

```diff
@@ -36,8 +36,8 @@ target 'WooCommerce' do
   pod 'Gridicons', '~> 1.0'
 
   # To allow pod to pick up beta versions use -beta. E.g., 1.1.7-beta.1
-  pod 'WordPressAuthenticator', '~> 1.26.0'
-  # pod 'WordPressAuthenticator', :git => 'https://github.com/wordpress-mobile/WordPressAuthenticator-iOS.git', :commit => ''
+  # pod 'WordPressAuthenticator', '~> 1.28.0-beta.2'
```

Comment:
Looking at the `Podfile.lock` history, Woo doesn't include the specific beta version. So when this is updated, it should be `1.28.0-beta`.